test/cpp/end2end/client_interceptors_end2end_test.cc

```diff
@@ -40,6 +40,10 @@
 #include <gtest/gtest.h>
 
 namespace grpc {
+
+namespace experimental {
+void TestOnlyResetGlobalClientInterceptorFactory();
```

Comment:
Prefer removing this forward decl and declaring in a header file instead.

Reply:
done